identifiers/country-kr/single_member_constituencies.csv

```diff
@@ -0,0 +1,253 @@
+id,name
+ocd-division/country:kr/metropolitan_city:26/ed:central_district_and_yeong_island,Central District and Yeong Island

```

Comment:
From https://en.wikipedia.org/wiki/South_Korean_Legislature_Constituencies

It sounds like the National Eleciton Commission handles the determination of the districts for South Korea's Legislature. Given that, I think the districts should hang off of country:kr instead of off of the individual admin area 1 OCD IDs.

Reply:
Done.